Main.java

```diff
@@ -0,0 +1,43 @@
+package ru.geekbrains.java;
+
+public class Main {
+    // задание 1
+    public static void main(String[] args) {
```

Comment:
How to run all the tasks?

Reply:
In the task for lesson 1, there is no need to make methods capable of running, because here they were mainly checked for correctness about syntax (it was said in the lesson).